### DISCUSSION TITLE
Android promo code option

### DISCUSSION THREAD

**Discussion by DanDance** [hyochan/react-native-iap#2170] [Q&A]
There is no option to use promo code as payment method. Google documentation says that there should be such an option during regular purchase flow. Promo code is set up in google (custom). But there's only "Add card" option.

Tried on 8.*, 10.* and 12.* versions. Tried make purchase with `requestPurchase` and `requestSubscription`. Tested via internal testing. Redmi note 7. 

What do I miss?

**Comment by DanDance** [comment]
Found out that there is limitation by countries to use promocodes
![image](https://user-images.githubusercontent.com/11865406/209521758-1d986859-b714-4a1c-92a5-49c01e8eb4cd.png)

**Comment by andresesfm** [comment]
This is handled by the native Android IAP: https://developer.android.com/google/play/billing/promo This library doesn't have control over it.
